saleor/graphql/api.py

```diff
@@ -313,21 +314,14 @@ class Mutations(graphene.ObjectType):
     menu_item_delete = MenuItemDelete.Field()
     menu_item_update = MenuItemUpdate.Field()
 
-    draft_order_create = DraftOrderCreate.Field()
-    draft_order_complete = DraftOrderComplete.Field()
-    draft_order_delete = DraftOrderDelete.Field()
-    draft_order_line_create = DraftOrderLineCreate.Field()
-    draft_order_line_delete = DraftOrderLineDelete.Field()
-    draft_order_line_update = DraftOrderLineUpdate.Field()
-    draft_order_update = DraftOrderUpdate.Field()
     order_fulfillment_cancel = FulfillmentCancel.Field()
     order_fulfillment_create = FulfillmentCreate.Field()
-    order_fulfillment_update_tracking = FulfillmentUpdateTracking.Field()
-    order_add_note = OrderAddNote.Field()
+    order_fulfillment_update_tracking = FulfillmentTrackingUpdate.Field()
+    order_add_note = OrderNoteAdd.Field()
```

Comment:
We have order_**add_note** but then we have **NoteAdd**.  We also have other cases below where we also have *ObjectName***Action** and not **Action**ObjectName. Plus, "OrderAddNote" sounds more correct English.

Reply:
I think ```CreateOrder``` sounds more correct than ```OrderCreate```, but that is the convention. Same applies to OrderNoteAdd. 

Should OrderNoteCreate be used here instead?

I will update the variable names, must have missed that.

Reply:
I think ```CreateOrder``` sounds more correct than ```OrderCreate```, but that is the convention. Same applies to OrderNoteAdd. 

Should OrderNoteCreate be used here instead?

I will update the variable names, must have missed that.